.github/workflows/quality-control-pr.yml

```diff
@@ -0,0 +1,31 @@
+name: Quality Control on pull request
+
+on:
+  pull_request:
+    schedule:
+      - cron: '30 5,17 * * *'
```

Comment:
Again, I think this cron schedule is unnecessary, and it even seems to have the wrong indentation. Please remove it